packages/cxl-ui/scss/cxl-course-card.scss

```diff
@@ -0,0 +1,197 @@
+:host {
+  display: block;
+  padding: var(--lumo-space-m) var(--lumo-space-l);
+  background: var(--lumo-tint);
+  border: 1px solid var(--lumo-contrast-10pct);
+  border-radius: var(--lumo-border-radius-s);
+  box-shadow: var(--lumo-box-shadow-xs);
+  break-inside: avoid;
+  transform: translateZ(0); // CSS columns @see https://stackoverflow.com/a/55110789/35946
+
+  .container {
+    display: none;
+  }
+}
+
+:host(:hover) {
+  border-color: var(--lumo-primary-color);
+}
+
+:host([hidden]) {
+  display: none;
+}
+
+:host(:first-child) {
+  margin-top: unset;
+}
+
+:host(:last-child) {
+  margin-bottom: unset;
+}
+
+:host([theme~="dark"]) {
+  background-color: var(--lumo-contrast);
+}
+
+/**
+ * CXL 2.0
+*/
+
+:host([theme~="cxl-course"]) {
+  position: relative;
+  height: max-content;
+  font-size: var(--lumo-font-size-s);
+
+  .container {
+    display: flex;
+    flex-direction: column;
+    gap: var(--lumo-space-s);
+  }
+
+  header {
+    display: flex;
+    align-items: start;
+    justify-content: space-between;
+    gap: var(--lumo-space-m);
+
+    .info {
+      display: flex;
+      flex-direction: column;
+      align-items: flex-start;
+      gap: var(--lumo-space-xs);
+      flex: 1 0 0;
+      align-self: stretch; 
+      max-width: calc(100% - var(--lumo-space-m) - 80px);
+      overflow: hidden;
+
+      .title {
+        color: #1A1A1A;
+        font-size: var(--lumo-font-size-xl);
+        font-family: Roboto;
+        font-style: normal;
+        font-weight: 700;
+        line-height: var(--lumo-line-height-xs);
+      }
+
+      .attributes {
+        display: flex;
+        padding: var(--lumo-space-s) 0;
+        align-items: flex-start;
+        gap: var(--lumo-space-s);
+        align-self: stretch;
+        color: var(--lumo-shade-60pct);
+      }
+    }
+
+    .instructor-image {
+      height: 92px;
+      width: 80px;
+      
+      img {
+        height: 80px;
+        border-radius: 100px;
+        overflow: hidden;
+      }
+    }
+
+    .tags span {
+      &:first-child, &.new {
+        color: var(--lumo-primary-color)
+      }
+    }
+  }
+
+  .tags {
+    display: flex;
+    gap: var(--lumo-space-s);
+    max-width: 100%;
+    overflow: hidden;
+
+    ::slotted(span):not(:first-child) {
+      overflow: hidden;
+      text-overflow: ellipsis;
+    }
+  }
+
+  .content {
+    .tags {
+      ::slotted(span) {
+        font-style: italic;
+      }
+    }
+  }
+
+  footer {
```

Comment:
Font size in this section should be the same as in other parts

Reply:
In the footer section, font-size is only adjusted for **::part(summary)** and **::part(toggle)** of the vaadin-details components header, to match the figma design as close as possible. Text content is not affected.